subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/rule/InHouseScenarioScreenRule.kt

```diff
@@ -0,0 +1,58 @@
+@file:Suppress("unused")
+package com.avito.android.rule
+
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ActivityScenario
+import androidx.test.platform.app.InstrumentationRegistry
+import com.avito.android.util.waitForAssertion
+import com.google.common.truth.Truth.assertThat
+import com.google.common.truth.Truth.assertWithMessage
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+abstract class InHouseScenarioScreenRule<A : Activity>(activityClass: Class<A>) : TestRule {
+
+    protected val androidInstrumentation: Instrumentation = InstrumentationRegistry.getInstrumentation()
+    protected val arguments: Bundle = InstrumentationRegistry.getArguments()
```

Comment:
Why do we need this property?
As I see, the rule doesn't use it itself and doesn't hide from children any information.

Reply:
Just copied it from the currently used `InHouseScreenRule`.
Did not found usages in the Avito project too and removed the property.